shared/js/media/jpeg_metadata_parser.js

```diff
@@ -305,7 +312,7 @@ function parseJPEGMetadata(file, metadataCallback, metadataError) {
 
     var total = count * typesize[type];
     var valueOffset = total <= 4 ? offset + 8 :
-      data.getUint32(offset + 8, byteorder);
+        data.getUint32(offset + 8, byteorder) + startOffset;
```

Comment:
Reading another patch I found the + 10 and thought the same. 

The reason I want to keep the +10 here is that the spec is vague about +10. It says "This tags records the offset from the start of the TIFF header to the position where the value itself is recorded." This is mostly "10" when it is an Exif block in a JFIF APP1 segment. So, short term it doesn't matter. Longer term, if we ever have a hardware that support raw files like some Android or Lumia do we'll be happy to support plain TIFF file (includes DNG which is TIFF/EP)

I'll update the PR to take that into account.